react-main/src/redux/game/actions.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export const selectSquare = position => {
```

Comment:
again, any function that only returns something can be written this way:

```
// bad
const foo = bar => {
  return something(bar); // or return { something: bar };
}

// good
const foo => bar => something(bar); // or ({ something: bar })
```

Please apply this to all other functions